docs/documentation/_data/deckhouse-alerts.yml

```diff
@@ -906,13 +957,19 @@ alerts:
       module: control-plane-manager
       edition: ce
       description: |
-        The etcd database size in use on instance `{{ $labels.instance }}` is less than 50% of the actual allocated disk space, indicating potential fragmentation
+        The etcd database size in use on instance `{{ $labels.instance }}` is less than 50% of the allocated disk space, indicating potential fragmentation. Additionally, the total storage size exceeds 75% of the configured quota.
```

Comment:
@maxinsky Please clarify me about "Additionally, the total storage size exceeds 75% of the configured quota".